plugin/pkg/admission/podpreset/admission.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	annotationPrefix = "podpreset.admission.kubernetes.io"
+	exclusionSuffix  = "PodPresetOptOut"
```

Comment:
@pmorie - ok.

then lets do the following:

`podpreset.alpha.admission.kubernetes.io/podpreset-{name}` = resource version
`podpreset.alpha.admission.kubernetes.io/opt-out = true`